backend/database/database.py

```diff
@@ -1,5 +1,4 @@
 from sqlalchemy import create_engine, text
-from sqlalchemy.ext.declarative import declarative_base
```

Comment:
Why do you delete that?

Reply:
It was an un-used import. do we still need this?